app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt

```diff
@@ -260,7 +260,8 @@ class MeetingViewModel(
 
   fun startMeeting() {
     if (!(state.value is MeetingState.Disconnected || state.value is MeetingState.Failure)) {
-      error("Cannot start meeting in ${state.value} state")
+      Log.d(TAG, "Not trying to join meeting when it's reconnecting.")
```

Comment:
This will stop the crash but we will not get to know what is the reason why this occured